src/stories/ui/form/Form.stories.tsx

```diff
@@ -0,0 +1,252 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Form } from '../../../sections/ui/form/Form'
+import { Col } from '../../../sections/ui/grid/Col'
+import { Row } from '../../../sections/ui/grid/Row'
+
+/**
+ * ## Description
+ * A form is a collection of HTML elements used to gather user input. Allows users to enter data, such as text, numbers,
+ * or file uploads, and submit it to a server for processing.
+ *
+ * ## Usage guidelines
+ * ### Dos
+ * - Input labels:
+ *   - Each input field should have a label
+ *   - Labels should be short and descriptive
+ *   - They should be placed above the input field
+ * - Input text:
+ *  - If you need to describe an input text you can use the `Form.Group.Text` component
+ *  - Add the text below the input
+ * - Select:
+ *  - First option should be the 'Select...' option
+ *  - The options should use the <option> tag
```

Comment:
`<option>` doesn't show up. Put it in backticks, maybe?

Reply:
fixed!